### DISCUSSION TITLE
Your Personal Account was automatically blocked because its usage exceeds the fair use policy

### DISCUSSION THREAD

**Discussion by quarkhadron** [vercel/community#3427] [Help]
### Summary

I tried deploying a simple interactive storytelling game. However, seem like the background music exceeded the bandwidth for Hobby Plan. I have deleted that project and require some assist for the steps to get account unblocked.

### Steps to Reproduce

-

**Comment by quarkhadron** [comment]
@amyegan Would you be able to assist me on this issue? 🙏🏼

**Comment by amyegan** [comment]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
